.github/workflows/knative-style.yaml

```diff
@@ -109,7 +109,7 @@ jobs:
           curl -sfL https://raw.githubusercontent.com/get-woke/woke/main/install.sh | sh -s -- -b "${TEMP_PATH}" "${WOKE_VERSION}" 2>&1
           echo '::endgroup::'
 
-          echo "::add-path::${TEMP_PATH}"
+          echo "${TEMP_PATH}" >> $GITHUB_PATH
```

Comment:
Same here?

Reply:
Correct it fixes the github action failures.

Reply:
https://github.com/knative/serving/commit/d2ec5468b2fd5c018d314eba24941e3f22331264 We need to do the same as in here.

Reply:
Got it 👍 

/lgtm